doc/desktop/Troubleshooting.md

```diff
@@ -0,0 +1,136 @@
+These are some common issues you may run into while setting up React Native Qt.
+
+## Initial setup issues
+
+### `npm install` hangs
+Downgrade to version 5.5.1: `npm install -g npm@5.5.1`.
+
+### `re-natal` missing
+Create a link:
+`ln -sf node_modules/re-natal/index.js re-natal`
+
+
+### `react-native run desktop` complaining about missing `qmldir`:
+```Command failed: ./build.sh -e "node_modules/react-native-i18n/desktop;node_modules/react-native-config/desktop;node_modules/react-native-fs/desktop;node_modules/react-native-http-bridge/desktop;node_modules/react-native-webview-bridge/desktop;modules/react-native-status/desktop"
+Error copying directory from "/path-to-status-react/node_modules/react-native/ReactQt/runtime/src/qmldir" to "/path-to-status-react/desktop/lib/React".
+make[2]: *** [lib/CMakeFiles/copy-qmldir] Error 1
+make[1]: *** [lib/CMakeFiles/copy-qmldir.dir/all] Error 2
+make: *** [all] Error 2
+```
+Can be solved by re-running `npm install react-native` which put the `ReactQt/runtime/src/qmldir` file back.
+
+### Missing web3 package issue
+
+After last upgrade of react-native-desktop to the v.0.53.3 of original react-native appeared some incompatibility between `react-native` and `web3` packages on npm install. Initially it installed usually fine, but after `react-native desktop` command execution `web3` package is get removed from `node_modules`. Manual install of web3 by `npm install web3` installs `web3` package, but removes `react-native` package. Workaround or solution?
+
+### Go problem
+```
+panic: runloop has just unexpectedly stopped
+
+goroutine 50 [running]:
+github.com/status-im/status-go/vendor/github.com/rjeczalik/notify.init.0.func1()
+        /path-to-status-react/desktop/modules/react-native-status/desktop/StatusGo/src/github.com/status-im/status-go/vendor/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:69 +0x79
+created by github.com/status-im/status-go/vendor/github.com/rjeczalik/notify.init.0
+        /path-to-status-react/desktop/modules/react-native-status/desktop/StatusGo/src/github.com/status-im/status-go/vendor/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:65 +0x4e
+events.js:183
+      throw er; // Unhandled 'error' event
+```
+Related to https://github.com/rjeczalik/notify/issues/139. Solution: re-run.
+
+## App issues
+
+### Node server crashing
```

Comment:
Probably we should clarify here, that "node" means not nodejs, but ethereum node :)

Reply:
Done, thanks for pointing that out.